programs/uxd/src/utils/calculate_depositories_sum_value.rs

```diff
@@ -0,0 +1,18 @@
+use anchor_lang::prelude::Result;
+
+use crate::error::UxdError;
+
+/**
+ * Compute the sum of one value for each known depositories
+ */
+pub fn calculate_depositories_sum_value(
```

Comment:
an utility function that is used everywhere in the rebalancing maths

Reply:
just making an existing function public:
https://github.com/UXDProtocol/uxd-program/pull/251/files#diff-8c17cb1da4f6615510c9e2f8d58fb63cc0a0b776dfa00dfa9e1b586049462dd1L245

Reply:
If functions becomes utility, we may make it less specific:

```
    pub fn calculate_depositories_sum_value(
        depositories_value: Vec<u64>,
   )
```